app/code/Ludmila/LSNewClasses/Model/ShowFiles.phtml

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Ludmila\LSNewClasses\Model;
+
+class ShowFiles
+{
+    public function show()
+    {
+        $path = realpath('/misc/apps/magento-226/app/code/Ludmila/LSNewClasses');
+
+//        $objects = new RecursiveIteratorIterator(new RecursiveDirectoryIterator($path), RecursiveIteratorIterator::SELF_FIRST);
+        $objects = new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator($path), \RecursiveIteratorIterator::SELF_FIRST);
+        foreach($objects as $name => $object){
+            echo "$name"." was modified on ".date('F d Y H:i:s',filemtime($name))."\n";
```

Comment:
It's not a good idea to echo html in a model. You can do echo only in templates, but it is not a template. There are two ways of correcting it:
1) return the array of files here (so the method should be renamed) and output them in the template
2) return string as a files list here (instead of echo just concatenate them in the result string and then return it);
I would prefer the first approach, as there shouldn't be html in models